index.js

```diff
@@ -84,13 +96,14 @@ module.exports = (api, options) => {
       }))
     }
 
-    if (isDevelopment) {
+    if (options.api === 'chrome' && isDevelopment) {
       const entries = { background: 'background' }
 
       if (pluginOptions.components.contentScript) {
         entries.contentScript = 'content-script'
       }
 
+      const ChromeExtensionReloader = require('webpack-chrome-extension-reloader')
```

Comment:
Awesome work and thanks for taking the time to contribute. For full transparency, I'm probably going to keep this import at the top of the file so that all dependencies are in one place and explicit. But I didn't feel that it was necessary to block your PR from getting merged to accomplish that.